elements/head.html

```diff
@@ -0,0 +1,39 @@
+<article class="article">
+  <header class="article-header">
+    <h2><code>&lt;head></code></h2>
```

Comment:
It is very strange that you are only escaping the opening bracket and not the closing bracket in all of these code blocks.

While I trust that it works in whatever browser you are developing against, I would not rely on that behavior across all browsers and all versions, and opt instead to explicitly encode the closing bracket as well:
```html
<h2><code>&lt;head&gt;</code></h2>
```